examples/lib/stories/bridge_libraries/flame_forge2d/utils/boxes.dart

```diff
@@ -51,6 +58,7 @@ class DraggableBox extends Box with DragCallbacks {
     required super.startPosition,
     required super.width,
     required super.height,
+    super.density = 1000,
```

Comment:
How come it needs a density of 1000, that seems a bit excessive?

Reply:
Thanks for the review. I set it to this value to decrease the oscillation, but after your comment, I tried to increase the `maxForce` and it helped as well. So I reverted the density to 10.

What do you think about the latest commits?

Reply:
Much better :+1: